report_stock_spreadsheet/LICENSE

```diff
@@ -0,0 +1,661 @@
+GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
You can remove this file.
You can use a comment with link like as template: https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3
And license key in manifest: https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L11